### DISCUSSION TITLE
demo/layout

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#878] [Announcements]
# demo/layout

布局包括: 导航栏 侧边栏 页脚 同时每个页面包含: 路径导航 标题和页面信息 TOC (文章标题列表) 贡献者、更新时间等页面元信息 评论 主题也带有以下元素: 夜间模式按钮 返回顶部按钮 打印按钮 你可以在主题选项和页面的 frontmatter 中自定义它们。

http://localhost:8080/demo/layout.html

**Comment by IOTWord** [comment]
123

**Comment by IndieCatGames** [comment]
good
